packages/engine-rn-web/package.json

```diff
@@ -40,7 +40,8 @@
         "html-webpack-harddisk-plugin": "1.0.2",
         "html-webpack-plugin": "4.5.1",
         "mini-css-extract-plugin": "0.8.0",
-        "webpack-bundle-analyzer": "3.5.1"
+        "webpack-bundle-analyzer": "3.5.1",
+        "react-native-web": "0.18.0"
```

Comment:
engine-rn-next should require react-native-web as well

Reply:
added